### PR TITLE
German translation

### PR DESCRIPTION
**Pull request by tayfuuun**
german translation added ;) 
have fun and please check my issue https://github.com/szTheory/exifcleaner/issues/86
It will be very nice when you add the option to remove the source.

### CODE REVIEW

**Comment by szTheory** [comment]
Thank you!

**Comment by tayfuuun** [comment]
> Thank you!

Thank you for this great tool.

File: .resources/strings.json [hunk 1]
```diff
@@ -19,130 +20,148 @@
...
+		"de": "# Exif bevor"
 	},
 	"table.header.exif-after": {
 		"en": "# Exif After",
 		"fr": "# Exif Après",
 		"pl": "# Exif po",
 		"ja": "# 処理後の Exif",
-		"es": "Exif después"
+		"es": "Exif después",
+		"de": "# Exif danach"
```
Comment (philippsandhaus) [inline .resources/strings.json:40]: besser: # Exif nachher

File: .resources/strings.json [hunk 2]
```diff
@@ -19,130 +20,148 @@
...
+		"de": "Ausgewählte Dateien"
 	},
 	"table.header.exif-before": {
 		"en": "# Exif Before",
 		"fr": "# Exif Avant",
 		"pl": "# Exif przed",
 		"ja": "# 処理前の Exif",
-		"es": "Exif antes"
+		"es": "Exif antes",
+		"de": "# Exif bevor"
```
Comment (philippsandhaus) [inline .resources/strings.json:32]: besser: # Exif vorher

File: .resources/strings.json (hunk 1 above)
Comment (tayfuuun) [reply .resources/strings.json:40]: > besser: # Exif nachher

thank you, need PR is created
https://github.com/szTheory/exifcleaner/pull/88

File: .resources/strings.json (hunk 2 above)
Comment (tayfuuun) [reply .resources/strings.json:32]: > besser: # Exif vorher

thank you, need PR is created
https://github.com/szTheory/exifcleaner/pull/88

**Pull request merged**